### ISSUE TITLE
Sodium and Iris Not Compatible With Embeddium

### ISSUE THREAD

**Issue by BMK2** [EnigmaticaModpacks/Enigmatica10#170]
### Modpack Version

1.11.0

### Describe your issue.

It seems like Embeddium was left in the latest list of mods but Sodium and Iris are not compatible with that mode. 

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by HeatherComputer** [comment]
Embeddium is not in the latest version, the one that added sodium / iris - see https://www.curseforge.com/minecraft/modpacks/enigmatica10/files/5814792/dependencies

**Comment by BMK2** [comment]
Yep, that's my fault. GD Launcher is still treating the 1.11 version as a beta version so it downloaded 1.10 first by default.
